test/unit/orm-adapters-memory-test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
+import { MemoryAdapter } from '../../lib/data/orm-adapters/memory';
+
+test.failing('MemoryAdapter > returns true after deleting an attribute', async (t) => {
+  t.plan(1);
```

Comment:
This isn't necessary - the test is sync, so the `t.true` line is guaranteed to execute. `t.plan` is primarily useful if you have async or conditional assertions that might not be reached if something is misbehaving, so therefore you want to tell `ava` to fail unless N assertions fired.